index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio | Abenezer Fekadu</title>
+  <link rel="stylesheet" href="./style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <div class="navbar-brand">Abenezer</div>
+    
+      
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
```

Comment:
- [ ] It is required that the hamburger of your page should be wrapped in a `<nav>` tag as this for semantics reason, and the right position for convention and user experience for mobile devices.

Reply:
@Qoosim 
The hamburger is already wrapped in the <nav>

Reply:
- [ ] At this moment, your hamburger is missing on the page. Kindly look at that and fix.

### Your Design
![your-hamburger](https://user-images.githubusercontent.com/26597432/213657633-66e351fe-c844-46f9-83fc-67b99cd81f66.png)

### Figma Design
![temp-hamburger](https://user-images.githubusercontent.com/26597432/213657668-92704ace-fc56-4151-80f0-7982d46acca9.png)